lib/nxdk/mount.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2019 Stefan Schmidt
+ *
+ * Licensed under the MIT License
+ */
+
+#ifndef __NXDK_MOUNT_H__
+#define __NXDK_MOUNT_H__
+
+#ifdef __cplusplus
+{
+#endif
+
+#ifndef __cplusplus
+#include <stdbool.h>
+#endif
+
+bool XIsDriveMounted (char driveLetter);
+bool XMountDrive (char driveLetter, char *path);
+bool XUnmountDrive (char driveLetter);
```

Comment:
Still torn about these prefixes: X is used in the original XDK for XGraphics, XAudio (I think?), XInput and maybe other APIs; this could lead to conflicts.
That way we don't have to mix MS and our APIs (as I'd like to implement XInput compatibility probably, but potentially also a new nxdk input layer).

Therefore I still consider another prefix.

We could consider `nx`/`nX` for **nx**dk or something different.
I don't have an ideal solution; I just want to raise concerns over the proposed `X` prefix.

Reply:
nx prefix works for me: It's unlikely to conflict with other libraries and it's pretty common to see such prefixes in console development kits.

A decision doesn't have to be made right now, but this shouldn't hold back this PR imo.

Perhaps it's worth leaving it as is for now, and creating an issue for a discussion, and once a decision is made, we can make all of nxdk consistent with a prefix. 

This might break some existing projects, but there aren't that many yet, better to do something like this sooner than later.

Reply:
I'm fine with the `nx` prefix, too. I've updated the PR to use it instead.